src/wallet/wallet.cpp

```diff
@@ -677,10 +689,10 @@ bool CWallet::EncryptWallet(const SecureString &strWalletPassphrase) {
     if (IsCrypted())
         return false;
 
-    CKeyingMaterial vMasterKey;
```

Comment:
I am strongly against renaming like that. It makes any core upgrade a lot harder. If there is a conflict with another variable introduced in this PR that variable needs renaming, not the original one

Reply:
ok will try to find a better solution.